forensics/replay.go

```diff
@@ -131,70 +131,64 @@ func (re *Replay) Block(height uint64) (*ReplayCapture, error) {
 func (re *Replay) Blocks(startHeight, endHeight uint64) ([]*ReplayCapture, error) {
 	var err error
 	var st *state.State
+
 	if startHeight > 1 {
 		// Load and commit previous block
-		block, err := re.explorer.Block(int64(startHeight - 1))
+		block, err := re.Explorer.Block(int64(startHeight - 1))
 		if err != nil {
-			return nil, err
+			return nil, errors.Wrap(err, "explorer.Block()")
 		}
 		err = re.blockchain.CommitBlockAtHeight(block.Time, block.Hash(), block.Header.AppHash, uint64(block.Height))
 		if err != nil {
-			return nil, err
-		}
-		// block.AppHash is hash after txs from previous block have been applied - it's the state we want to load on top
-		// of which we will reapply this block txs
-		st, err = re.State(startHeight - 1)
-		if err != nil {
-			return nil, err
-		}
-	} else {
```

Comment:
@silasdavis isn't this required for a non-restored chain?

Reply:
@gregdhill no, when I wrote it I thought it was, when we pull state at height 0, we get state version 1, which is the initial commit containing the genesis state as originally laid down. So this is better than us recreating it here and doesn't require a special code path.